report/26_Process-Development-Process-&-Tools.md

```diff
@@ -1,2 +1,52 @@
-Applied development process and tools supporting it
-For example, how did you use issues, Kanban boards, etc. to organize open tasks
+### Applied development process and tools supporting it
+
+Throughout this course, two tools, Github and Discord, has been used to manage the project and communication.
+Two tools have been used, to differentiate between synchronous and asynchronous communication.
+That is, Discord is used to communicate directly and informally to allow for fast and efficient communication,
+whereas GitHub is used to organize our work in a way that can easily be searched through and understood at a later date.
+
+GitHub has been the ideal tool for the majority of our project management, because both the code, git commits, and corresponding issues coexist and can easily be cross-referenced.
+Furthermore, GitHub has the concept of "Pull requests" (PR), which introduces a way to do formal review policy for new code, before merging it with the master branch.
+These pull requests can also be linked with the aforementioned issues, which makes it easy to see which code-changes fixes a given issue.
+
+The Github workflow starts by creating the issues. The actual content of the issues is often agreed upon verbally or dictated by the homework for the course. The issues are then augmented with relevant metadata, such as description, tags, milestones, assignees, and comments. Most importantly, we use a tag for all mandatory work, so that we know what needs to be completed.
+Occasionally, when many tasks need to be solved in a certain timeframe, such as when writing this report, we make a "milestone" in GitHub.
+In this, we can attach all relevant issues, set a due date and afterward track the combined progress for all tasks.
```

Comment:
Reword this entire sentence. Remove we, start with explaining that its a postive of milestones.